src/upper-level-components/Pages.js

```diff
@@ -19,6 +19,15 @@ const Pages = {
     },
     loginBadge: false,
   },
+  google: {
+    displayName: 'Google',
+    paths: ['/google'],
+    authorizations: {
+      clickup: false,
```

Comment:
Now that you've removed the need for a ClickUp token for Button Registration, Twilio Number Purchasing, and Renamer, do you think that those pages can have `clickup: false` now?

Reply:
Good point! I'll do this!

Reply:
Renamer still needs clickup, but Button Registration and Twilio Number Purchasing don't. I've set clickup to false for those pages.